src/view-backend-exportable-fdo-egl.cpp

```diff
@@ -247,9 +247,13 @@ class ClientBundleEGL final : public ClientBundle {
 
     void releaseImage(struct wpe_fdo_egl_exported_image* image)
     {
-        if (image->bufferResource)
-            viewBackend->releaseBuffer(image->bufferResource);
-        else
+        if (!image)
+            return;
+        if (image->exported) {
+            image->exported = false;
```

Comment:
I understand this change restores the first logic (`exported` safe-guard ) implemented in the original commit https://github.com/Igalia/WPEBackend-fdo/commit/fb897d2972a0bbc4d0157512f3da281ec7be917f#diff-a2ff2fb4cb225868a0a9b4ae69769ab3efc9e6fbb06d26885c123582bc7a3a0bR198 which it was not problematic at all. This looks good.